stable/opsportal/requirements.yaml

```diff
@@ -1,5 +1,5 @@
 dependencies:
-- name: kommander
-  version: 0.2.7
-  repository: https://mesosphere.github.io/charts/stable
-  condition: kommander.enabled, global.kommander.enabled
\ No newline at end of file
+  - name: kommander-ui
```

Comment:
kommander no longer needs to be a requirement for opsportal since we now have our own kommander ui chart, thats the only requirement we actually need here (**is that correct?**)

Reply:
This would lead to the `kommander-ui` chart to be installed twice in Konvoy since now both, `kommander` and `opsportal` depend on `kommander-ui`.

Reply:
thats true, but current status is even worse: opsportal depends on kommander, which means everything (KCL, thanos, karma, UI and more) is installed twice… 

after this PR is merged, I want to extract ui into its own kubeaddon so that neither opsportal nor kommander actually depend on the UI. need to clarify a couple things before I'll do that tho

Reply:
thoughts / input on that idea highly appreciated btw, I'm still not not completely confident with how these charts actually work in detail